packages/grid/src/stories/grid.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import { Grid } from '../grid';
+
+export default {
+	title: 'Grid',
+	component: Grid,
+	tags: [ 'autodocs' ],
+	parameters: {
+		layout: 'centered',
+	},
+};
+
+function Card( {
+	style,
+	color,
+	children,
+}: {
+	style?: React.CSSProperties;
+	color: string;
+	children: React.ReactNode;
+} ) {
+	return (
+		<div
+			key="a"
+			style={ {
+				backgroundColor: color,
+				color: 'white',
+				padding: '20px',
+				display: 'flex',
+				alignItems: 'center',
+				justifyContent: 'center',
+				...style,
+			} }
+		>
+			{ children }
+		</div>
+	);
+}
+
+/**
+ * Basic usage example of the Grid component
+ */
+export const Default = () => {
+	const layout = [
+		{ key: 'a', x: 0, y: 0, width: 1 },
+		{ key: 'b', x: 1, y: 0, width: 3 },
+		{ key: 'c', x: 4, y: 0, width: 1 },
+	];
+
+	return (
+		<Grid layout={ layout } cols={ 6 } gap="10px" rowHeight="100px">
```

Comment:
We've found it useful to keep the `Default` story showing the most effortlessly default state, as far as it's practical in real life. In this case, that means leaving the `gap` and `rowHeight` undefined so it defaults to the actual default values in code.